src/static/js/modules/external-site-redirect.js

```diff
@@ -16,7 +16,7 @@ function init() {
     // Futher explanation can be viewed
     // at https://regex101.com/r/xT7sL5/2.
     var externalURLArray =
-        ( /(https?:\/\/(?:www\.)?(?!.*gov)(?!localhost).*)/g )
+        ( /(https?:\/\/(?:www\.)?(?!.*gov)(?!(?!content\.)localhost).*)/g )
```

Comment:
Try `(https?:\/\/(?:www\.)?(?!.*gov)(?!(content\.)?localhost).*)`. Further explanation can be viewed at https://regex101.com/r/xT7sL5/2.